fieldservice/models/fsm_order.py

```diff
@@ -299,6 +299,41 @@ def _onchange_template_id(self):
             self.scheduled_duration = self.template_id.hours
             self.copy_notes()
 
+    def _compute_workers_to_request(self):
+        email = ""
+        for person in self.person_ids:
+            if person.email:
+                email += person.email + ','
+        email = email[:-1]
+        self.workers_to_request = email
+
+    def _compute_workers_not_selected(self):
+        email = ""
+        if self.person_id:
+            for person in self.person_ids:
+                if (person.email and (person != self.person_id)):
+                    email += person.email + ','
+        email = email[:-1]
+        self._compute_workers_not_selected = email
+
+    def _compute_outage_list(self):
+        rtype = self.env['res.partner.relation.type'].\
```

Comment:
@osi-scampbell `fieldservice` does not depend on the module providing this model. It will not work.

Reply:
I see, I think it should be moved to pavlov_media_fieldservice then

Reply:
Yes please.

Reply:
Done